docs/tutorial/fastapi/update.md

```diff
@@ -222,7 +222,7 @@ So, we would use that value and upate the `age` to `None` in the database, **jus
 
 Notice that `age` here is `None`, and **we still detected it**.
 
-Also that `name` was not even sent, and we don't *accidentaly* set it to `None` or something, we just didn't touch it, because the client didn't sent it, so we are **pefectly fine**, even in these corner cases. ✨
+Also that `name` was not even sent, and we don't *accidentaly* set it to `None` or something, we just didn't touch it, because the client didn't send it, so we are **pefectly fine**, even in these corner cases. ✨
```

Comment:
```suggestion
Also that `name` was not even sent, and we don't *accidentally* set it to `None` or something, we just didn't touch it, because the client didn't send it, so we are **perfectly fine**, even in these corner cases. ✨
```